src/api/session/notary/Notary.cpp

```diff
@@ -318,24 +318,18 @@ auto Notary::GetPrivateMint(
     // Modifying the private version may invalidate the public version
     seriesMap.erase(PUBLIC_SERIES);
 
-    auto& output = [&]() -> auto&
-    {
-        if (auto it = seriesMap.find(seriesID); seriesMap.end() != it) {
-
-            return it->second;
-        }
-
-        auto [it, added] = seriesMap.emplace(seriesID, *this);
-
+    otx::blind::Mint* output{nullptr};
```

Comment:
I suggest the same trick with c++17 try_emplace, which will give you a single liner, preserving a reference rather than a pointer.